pyad/adobject.py

```diff
@@ -352,7 +352,7 @@ def get_user_account_control_settings(self):
         d = {}
         auc = self.get_attribute('UserAccountControl',False)
         for key, value in ADS_USER_FLAG.items():
-            d[key] = True if auc & value == value else False
+            d[key] = auc & value == value

```

Comment:
I'm not sure if I understand this change. This two lines should be identical.

Reply:
> I'm not sure if I understand this change. This two lines should be identical.

They are, it's just redundant; since it's already a bool, there's no need to say "True if foo_bool else False"

Reply:
Oh! Yes. Sorry I thought the diff added the unnecessary part. 🤦‍♂️